packages/web/app/src/components/policy/policy-settings.tsx

```diff
@@ -54,7 +54,7 @@ function PolicySettingsListForm({
   saving?: boolean;
   rulesInParent?: string[];
   error?: string;
-  onSave: (values: SchemaPolicyInput, allowOverrides: boolean) => Promise<void>;
+  onSave?: (values: SchemaPolicyInput, allowOverrides: boolean) => Promise<void>;
```

Comment:
I overloaded onSave to indicate whether or not this is readonly. If onSave is undefined, then this is considered readonly and all settings will be disabled.

I'm open to adding a separate attribute instead of that's preferred.